AUTHORS

```diff
@@ -0,0 +1 @@
+Ahmed Ratnani, Mohamed Jalal Maaouni
```

Comment:
See my comment [here](https://github.com/pyccel/IGA-Python/pull/9#discussion_r1970045013): please format this file as standard with one author per line.

Also a newline character is missing at the end of the file.

Reply:
Fixed

Reply:
You should add your name to the AUTHORS